main/FAB/index.tsx

```diff
@@ -46,10 +53,12 @@ function FloatingActionButtons<Item extends FABItem = FABItem>({
 }: FABProps<Item> & {
   theme: DoobooTheme;
 }): ReactElement {
+  const GAP = 15;
```

Comment:
How about moving this to `styles`?

Reply:
That is a good idea thanks!

Reply:
@yujong-lee 
I reflected this review thanks.